api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -300,8 +337,40 @@ def __init__(
         """
         Construct a new ``ExperimentDataPipe``.
 
+        Args:
+            experiment:
+                The SOMA ``Experiment`` from which to read data.
+            measurement_name:
+                The name of the SOMA ``Measurement`` to read. Defaults to "raw".
+            X_name:
+                The name of the X layer to read. Defaults to "X".
+            obs_query:
+                The query used to filter along the ``obs`` axis. If not specified, all ``obs`` and ``X`` data will
+                be returned, which can be very large.
+            var_query:
+                The query used to filter along the ``var`` axis. If not specified, all ``var`` columns (genes/features)
+                will be returned.
+            obs_column_names:
+                The names of the ``obs`` columns to return. The ``soma_joinid`` index "column" does not need to be
+                specified and will always be returned. If not specified, only the ``soma_joinid`` will be returned.
+            batch_size:
+                The number of rows of ``obs`` and ``X`` data to return in each iteration. Defaults to 1. A value of 1
+                will result in Tensors of rank 1 being returns (a single row); larger values will result in Tensors of
+                rank 2 (multiple rows).
+            sparse_X:
```

Comment:
Mostly a nitpick, but maybe `return_sparse_X` is a better name?

Reply:
done!